NewEssentials/Chat/IBroadcastingService.cs

```diff
@@ -0,0 +1,12 @@
+using System.Threading.Tasks;
+using OpenMod.API.Ioc;
+
+namespace NewEssentials.Chat
+{
+    [Service]
+    public interface IBroadcastingService
+    {
+        public bool IsActive { get; set; }
```

Comment:
XML Documentation would be nice but not necessary. We can add a TODO here and document it later if you want.

Reply:
Will push now